tests/test_custom_models.py

```diff
@@ -1005,7 +1055,7 @@ def test_disable_adapters(self, test_name, model_id, config_cls, config_kwargs):
         else:
             rtol, atol = 1e-5, 1e-8
         assert not torch.allclose(outputs_before, outputs_after, rtol=rtol, atol=atol)
-        assert torch.allclose(outputs_before, outputs_disabled)
+        assert torch.allclose(outputs_base, outputs_disabled)
```

Comment:
If `init_weights` is implemented for VB-LoRA, this can be reverted, right?

Reply:
Yes, but we are having difficulty implementing `init_weights` in the same way as LoRA.

Reply:
I followed your first suggestion in the comment: 
1. I removed `init_weights`.
2. In tests, I set the vector bank to zero to make VB-LoRA as an identity operation. And initialized the vector bank if training is involved in the test.

I recovered this line back.